packages/js/src/settings/components/search.js

```diff
@@ -29,96 +16,137 @@ const Search = () => {
 	// eslint-disable-next-line no-unused-vars
 	const [ isOpen, , , setOpen, setClose ] = useToggleState( false );
 	const [ query, setQuery ] = useState( "" );
-	const items = useSelectSettings( "selectFlatSearchIndex" );
+	const searchIndex = useSelectSettings( "selectSearchIndex" );
+	const queryableSearchIndex = useSelectSettings( "selectQueryableSearchIndex" );
 	const [ results, setResults ] = useState( [] );
 	const ariaSvgProps = useSvgAria();
-	const inputRef = useRef();
-	const navigate = useNavigate();
+	const inputRef = useRef( null );
 
-	const handleChange = useCallback( item => {
-		navigate( `${ item.route }#${ item.fieldId }` );
+	const handleNavigate = useCallback( () => {
```

Comment:
Something with this function is missing. Screen goes blank when I close the modal and I'm getting an error in the console:
`Error: A <Transition /> is used but it is missing a `show={true | false}` prop.`